nucleus/modelci/eval_functions/available_eval_functions.py

```diff
@@ -0,0 +1,81 @@
+from typing import List, Dict, Union, Type
+
+from nucleus.modelci.eval_functions.base_eval_function import BaseEvalFunction
+from nucleus.modelci import EvalFunctionEntry
+
+MEAN_AVG_PRECISION_NAME = "mean_average_precision_boxes"
+IOU_NAME = "IOU"
+
+
+class IoU(BaseEvalFunction):
+    """Intersection over union for all bounding boxes"""
+
+    @classmethod
+    @property
+    def name(self) -> str:
+        return "IOU"
+
+
+class MeanAveragePrecisionForBoxes(BaseEvalFunction):
+    """Mean average precision for boxes fitting the provided parameters"""
+
+    @classmethod
+    @property
+    def name(self) -> str:
+        return "mean_average_precision_for_boxes"
+
+
+EvalFunction = Union[Type[IoU], Type[MeanAveragePrecisionForBoxes]]
```

Comment:
Why not just use `BaseEvalFunction`?

Reply:
MyPy wasn't happy with the base class for some reason 🤷‍♂️